doc/api/errors.md

```diff
@@ -1313,6 +1313,18 @@ While using `N-API`, a constructor passed was not a function.
 
 While using `N-API`, `Constructor.prototype` was not an object.
 
+<a id="ERR_NAPI_INVALID_TYPEDARRAY_ALIGNMENT"></a>
+### ERR_NAPI_INVALID_TYPEDARRAY_ALIGNMENT
+
+While calling `napi_create_typedarray()`, a given `offset` was not a multiple of
+size of element of given type.
```

Comment:
Maybe change "size of element of given type" to "the element size"?

Reply:
Done